.github/workflows/validators_join_and_leave.yaml

```diff
@@ -0,0 +1,21 @@
+name: Validator Join and Leave Test
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
```

Comment:
below this add 

```
pull_request:
  paths: 
    - ".github/workflows/validator_join_and_leave.yaml
```

This will actually run this job on any PR touching this file!